WordPress/Classes/Apps/Reader/ReaderTabViewController.swift

```diff
@@ -108,7 +108,7 @@ final class ReaderTabViewController: UITabBarController, UITabBarControllerDeleg
     }
 
     private func makeNotificationsViewController() -> UIViewController {
-        let notificationsVC = UIStoryboard(name: "Notifications", bundle: nil)
+        let notificationsVC = UIStoryboard(name: "Notifications", bundle: Bundle(for: Self.self))
```

Comment:
Required to avoid a crash due to the storyboard in Reader being part of Keystone, not the app bundle

![image](https://github.com/user-attachments/assets/c70432e8-76d4-4c57-8369-15135bfaf93e)

Reply:
https://github.com/wordpress-mobile/WordPress-iOS/pull/24464 follows up to this and addresses all such accesses for the notifications storyboard.